kit/buildingblocks/starterkit/buildingblock/main.tf

```diff
@@ -0,0 +1,33 @@
+data "azurerm_subscription" "current" {}
+data "azuread_client_config" "current" {}
+
+# note: it's important that all other azure resources transitively depend on this role assignment or else they will fail
+resource "azurerm_role_assignment" "starterkit_deploy" {
```

Comment:
d: we should probably document this more thoroguhly in the Kit readme. 

It's "one" possible access strategy. There is also a solution pattern of "brokering" access to tenants where the CFT owns the role definitions and granting those roles and e.g. the DevOps Toolchain team needs to request that BB dependency